### ISSUE TITLE
Zotero: set up

### ISSUE THREAD

**Issue by ErwinKomen** [ErwinKomen/RU-lilac#28]
Provide the functionality to work with a Zotero group (account) and keep 'stuff' updated.

Relevant issues are: #3 , #4

**Comment by ErwinKomen** [comment]
See Tools > Synchronisation, which makes use of seeker/synclilac.html, and that in tern uses seeker/views_api.py with `synctype = "zotero"`
Next, this assumes a correct working of `Litref.sync_zotero()`, which should be checked manually first!

**Comment by ErwinKomen** [comment]
1. Okay, I figured out that two Information Items must be specified:
  `zotero_libraryid` - the numerical identifier of the Zotero group for LILAC (all references that have anything to do with LILAC should be in that group)
  `zotero_apikey` - the key for the LILAC group that should be created by the PI

1. I also double checked synchronization and emended code where needed (it was needed in some places). It should now basically work. 
1. And I checked whether it is possible to add references to a manuscript: it is now.
